zerver/tests/test_reactions.py

```diff
@@ -367,3 +368,462 @@ def test_remove_event(self):
         self.assertEqual(event['op'], 'remove')
         self.assertEqual(event['emoji_name'], 'smile')
         self.assertEqual(event['message_id'], pm_id)
+
+class DefaultEmojiReactionTests(ZulipTestCase):
+    def post_reaction(self, reaction_info, message_id=1, sender='hamlet'):
+        # type: (Dict[str, str], int, str) -> HttpResponse
+        reaction_info['reaction_type'] = 'unicode_emoji'
+        sender = self.example_email(sender)
+        result = self.client_post('/api/v1/messages/%s/reactions' % (message_id,),
+                                  reaction_info,
+                                  **self.api_auth(sender))
+        return result
+
+    def delete_reaction(self, reaction_info, message_id=1, sender='hamlet'):
+        # type: (Dict[str, str], int, str) -> HttpResponse
+        reaction_info['reaction_type'] = 'unicode_emoji'
+        sender = self.example_email(sender)
+        result = self.client_delete('/api/v1/messages/%s/reactions' % (message_id,),
+                                    reaction_info,
+                                    **self.api_auth(sender))
+        return result
+
+    def get_message_reactions(self, message_id, emoji_code, reaction_type):
+        # type: (int, Text, Text) -> List[Reaction]
+        message = Message.objects.get(id=message_id)
+        reactions = Reaction.objects.filter(message=message,
+                                            emoji_code=emoji_code,
+                                            reaction_type=reaction_type,
+                                            )
+        return reactions.all()
```

Comment:
`return list(reactions)` would be more standard here.

Reply:
Also, I didn't clean this one up, but I think we could share this function with the other reactions test class that has the same code (by having a `EmojiReactionTestBase` class)